app/models/image.rb

```diff
@@ -0,0 +1,4 @@
+class Image < ActiveRecord::Base
+  belongs_to :image_set
+  belongs_to :image_type
+end
```

Comment:
@redyaffle  I'd suggest validating that the image has a URL. Did you see the `s3-upload-rebased` branch I created? I also created an image model there and started working on the API. We can merge this branch first and I'll remove some stuff from that branch later, but I just wanted to suggest that we port over that [valdation](https://github.com/201-created/lion-guardians-api/compare/s3-upload-rebased#diff-69e1773ce93d7ede3d3d0c7f4e6c7024R2) and also the [unit tests](https://github.com/201-created/lion-guardians-api/compare/s3-upload-rebased#diff-31fdbb7677a4de0d6fbce02d2f1120c7R1).